proto/buf.lock

```diff
@@ -8,12 +8,12 @@ deps:
   - remote: buf.build
     owner: cosmos
     repository: cosmos-sdk
-    commit: 9e88ecdd390f419081b79a917958d0f9
+    commit: 4e1072d2e34943e79c5699c02aad7066
```

Comment:
Let's not update references here without a good reason. The commit points to a tag on [buf](https://buf.build/cosmos/cosmos-sdk/commits/5485fb0c87440a36a96da6e482154600e2c0b48c) that refers to commit [`5485fb0c87440a36a96da6e482154600e2c0b48c`](https://github.com/cosmos/cosmos-sdk/commit/5485fb0c87440a36a96da6e482154600e2c0b48c
) on main. 
The sdk version [v0.45.12](https://github.com/cosmos/cosmos-sdk/releases/tag/v0.45.12) used in go mod is older and on a different branch.
This will get easer with 47+ as they set a good tag on buf, now